frontend/packages/console-app/src/components/quick-starts/DrawerComponent.tsx

```diff
@@ -10,12 +10,12 @@ import {
   DrawerContentBody,
   Title,
 } from '@patternfly/react-core';
-import { GuidedTourItem, GuidedTourState } from './utils/guided-tour-typings';
-import './GuidedTourDrawer.scss';
+import { QuickStartItem, QuickStartState } from './utils/quick-start-typings';
+import './QuickStartDrawer.scss';
 
 type DrawerComponentProps = {
-  guidedTour: GuidedTourItem;
-  guidedTourState?: GuidedTourState;
+  QuickStart: QuickStartItem;
+  QuickStartState?: QuickStartState;
```

Comment:
props should follow camelCase naming
```suggestion
  quickStart: QuickStartItem;
  quickStartState?: QuickStartState;
```